wagtail/tests/test_management_commands.py

```diff
@@ -722,6 +722,13 @@ def test_purge_revisions_with_date_cutoff(self):
         # revision is now older than 30 days, so should be deleted
         self.assertRevisionNotExists(old_revision)
 
+    def test_purge_revisions_protected_error(self):
+        revision = self.object.save_revision()
```

Comment:
We need to call `save_revision()` twice and check that both revisions (especially the first revision) are not purged, because otherwise the revision will not be removed not because it's protected, but rather because it's the latest revision.